decidim/decidim
# Don't add the slug of the space in some links

#### :tophat: What? Why?
This PR makes sure there are no URL that is displaying the participatory space slug as query string. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11176 

#### Testing
1. Visit any participation space show page 
2. Login as admin 
3. Click edit button in the frontend 
4. You should be redirected to admin, not having the `<participatory_space>_slug` argument. 

:hearts: Thank you!
